po/fr.po

```diff
@@ -0,0 +1,296 @@
+msgstr "Général"
+
+#: Tab.cpp:48 shortcuts-dialog.blp:102 window.blp:16
+msgid "JavaScript Console"
+msgstr "Console JavaScript"
+
+#: Application.cpp:66
+msgid ""
+"Ladybird is a browser based on LibWeb web engine and LibJS JavaScript "
+"engine, developed by a large team of contributors as a part of the "
+"SerenityOS project."
+msgstr ""
+"Ladybird est un navigateur basé sur le moteur de rendu LibWeb et la bibliothèque JavaScript "
+"LibJS, développé par une équipe de contributeurs appartenant au projet SerenityOS."
+
+#: window.blp:218
+msgid "Main Menu"
+msgstr "Menu Principal"
+
+#: window.blp:145 window.blp:282
+msgid "Navigate Back"
+msgstr "Naviguer vers la page précédente"
+
+#: window.blp:157 window.blp:288
+msgid "Navigate Forward"
+msgstr "Naviguer vers la page suivante"
+
+#: shortcuts-dialog.blp:38
+msgid "Navigate back"
+msgstr "Page précédente"
+
+#: shortcuts-dialog.blp:42
+msgid "Navigate forward"
+msgstr "Page suivante"
+
+#: shortcuts-dialog.blp:35
+msgid "Navigation"
+msgstr "Navigation"
+
+#: window.blp:175
+msgid "New Tab"
+msgstr "Nouvel Onglet"
+
+#: window.blp:11 window.blp:43
+msgid "New _Incognito Window"
+msgstr "Nouvelle Fenêtre de Navigation _Privée"
+
+#: shortcuts-dialog.blp:17
+msgid "New incognito window"
+msgstr "Nouvelle fenêtre privée"
+
+#: Window.cpp:101 shortcuts-dialog.blp:59
+msgid "New tab"
+msgstr "Nouvel onglet"
+
+#: shortcuts-dialog.blp:13
+msgid "New window"
+msgstr "Nouvelle fenêtre"
+
+#: shortcuts-dialog.blp:67
+msgid "Next tab"
+msgstr "Onglet suivant"
+
+#: Window.cpp:236 Tab.cpp:194
+msgid "OK"
+msgstr "OK"
+
+#: shortcuts-dialog.blp:21
+msgid "Open file"
+msgstr "Ouvrir un fichier"
+
+#. Translators: this is a menu item for pinning a tab.
+#: window.blp:26
+msgid "P_in"
+msgstr "_Épingler"
+
+#: shortcuts-dialog.blp:71
+msgid "Previous tab"
+msgstr "Onglet précédent"
+
+#: shortcuts-dialog.blp:29
+msgid "Quit"
+msgstr "Quitter"
+
+#: window.blp:169 window.blp:294
+msgid "Reload Page"
+msgstr "Recharger la Page"
+
+#: shortcuts-dialog.blp:46
+msgid "Reload current page"
+msgstr "Actualiser"
+
+#: shortcuts-dialog.blp:89
+msgid "Reset zoom"
+msgstr "Taille réelle"
+
+#: window.blp:236
+msgid "Restore Zoom"
+msgstr "Taille réelle"
+
+#: Application.cpp:49
+msgid "Sam Atkins"
+msgstr "Sam Atkins"
+
+#. Translators: this is the placeholder text for the main location entry.
+#: window.blp:188
+msgid "Search or enter a URL to open…"
+msgstr "Rechercher ou saisir une adresse…"
+
+#: Application.cpp:57
+msgid "SerenityOS developers"
+msgstr "Développeurs SerenityOS"
+
+#: Application.cpp:70
+msgid "SerenityOS website"
+msgstr "Site web de SerenityOS"
+
+#. Translators: Сергей Бугаев in Cyrillic
+#: Application.cpp:44
+msgid "Sergey Bugaev"
+msgstr "Sergey Bugaev"
+
+#: shortcuts-dialog.blp:8
+msgid "Shortcuts"
+msgstr "Raccourcis"
+
+#: shortcuts-dialog.blp:25
+msgid "Show shortcuts"
+msgstr "Afficher les raccourcis"
+
+#: window.blp:212
+msgid "Tab Overview"
+msgstr "Aperçu de l‘Onglet"
```

Comment:
Same here: should the O be lowercase? (I don't know, maybe it shouldn't! Just bringing your attention to this)